README.org

```diff
@@ -44,6 +44,9 @@ Just make sure you have [[https://github.com/rust-analyzer/rust-analyzer][rust-a
 *** Install
 #+BEGIN_SRC bash
 git clone https://github.com/kitallis/dotemacs.git ~/.emacs.d
+# or
+git clone https://github.com/deobald/dotemacs.git
+ln -s /home/<you>/work/deobald/dotemacs ~/.emacs.d
```

Comment:
why clone wherever and then symlink, the first one don't work?

Reply:
the first one works fine, i just don't do it that way. you can remove those 2 lines if you want (maintainer edits are on)